django_setup_configuration/configuration.py

```diff
@@ -34,12 +34,12 @@ def is_enabled(self) -> bool:
         """
         Hook to switch on and off the configuration step from env vars
 
-        By default all steps are enabled
+        By default all steps are disabled
         """
         if not self.config_settings.enable_setting:
             return True
 
-        return getattr(settings, self.config_settings.enable_setting, True)
+        return getattr(settings, self.config_settings.enable_setting, False) or False
```

Comment:
Is the default value false not redundant? As in the the or will be checked if it is None or False

Reply:
If default value is not present then in case the setting is missing the exception would be raised
So default = `False` helps not to raise an exception